api/apps/api/src/modules/scenario-specification/__tests__/determine-features/fixtures.ts

```diff
@@ -0,0 +1,119 @@
+import { v4 } from 'uuid';
+import {
+  FeatureConfigInput,
+  ScenarioSpecification,
+} from '../../scenario-specification';
+import { SpecificationOperation } from '../../specification-operation';
+import { SpecificationPublished } from '../../specification-published.event';
+import { SpecificationActivated } from '../../specification-actived.event';
+
+export const getFixtures = () => {
+  const scenarioId = v4();
+  const baseFeatureId = v4();
+  const stratificationFeatureId = v4();
+  const nonCalculatedFeature = v4();
+
+  const scenarioSpec = new ScenarioSpecification(scenarioId, undefined, {
+    id: v4(),
+    draft: true,
+    config: [],
+    activated: false,
+  });
+
+  const splitConfig: FeatureConfigInput = {
+    operation: SpecificationOperation.Split,
+    baseFeatureId: baseFeatureId,
+  };
+
+  const stratificationConfig: FeatureConfigInput = {
+    operation: SpecificationOperation.Stratification,
+    baseFeatureId: baseFeatureId,
+    againstFeatureId: stratificationFeatureId,
+  };
+
+  const featuresFromSplit = [
+    {
+      id: v4(),
+      calculated: true,
+    },
+    {
+      id: v4(),
+      calculated: true,
+    },
+  ];
+
+  const featuresFromStratificationSomeCalculated = [
+    {
+      id: v4(),
+      calculated: true,
+    },
+    {
+      id: nonCalculatedFeature,
+      calculated: false,
+    },
+  ];
+
+  const featuresFromStratificationAllCalculated = [
+    {
+      id: v4(),
+      calculated: true,
+    },
+    {
+      id: v4(),
+      calculated: true,
+    },
+  ];
+
+  return {
+    scenarioSpec,
+    GivenDraftWasSubmitted: () => {
+      scenarioSpec.createCandidate([splitConfig, stratificationConfig], true);
+    },
+    GivenCreatedWasSubmitted: () => {
+      scenarioSpec.createCandidate([splitConfig, stratificationConfig], false);
+    },
```

Comment:
those given doesn't sound well in my mind with actions taken on the aggregate. If aggregate uses the domain language, why do fixtures need to translate it?